scripts/update-tiledb-version.py

```diff
@@ -103,6 +103,10 @@ def main(args):
     filepath = f"{os.path.dirname(__file__)}/../apis/r/tools/get_tarball.R"
     update_version(filepath, new_version, new_hash, update_sha=False)
 
+    # update CI version
+    filepath = f"{os.path.dirname(__file__)}/../.github/workflows/python-so-copying.yml"
```

Comment:
But this file won't exist after #2211 is merged, right? I'm not following the logic of backporting this commit. If it is simply a bot that backports every commit that lands on main, I think this can be closed

Reply:
@jdblischak please see #2210 

Yes, I do have multiple PRs in flight in multiple branches -- I'll resolve this bookkeeping issue -- freeing you up to work on other things -- I'll clean up from here

Reply:
Works for me. Thanks!